components/nds-column-list/src/ColumnList.tsx

```diff
@@ -25,6 +25,7 @@ export const ColumnList: React.FC<ColumnListProps> = (
 			]
 				.filter(Boolean)
 				.join(" ")}
+			data-component={`column-list--${plain}`}
```

Comment:
Could do with replacing this with a ternary expression if that's ok; what I mean by this is to replace `column-list--${plain}` with `column-list${plain? `--plain` : ""}`

The reason for this is because plain is an optional prop, so if it's omitted then the prop is undefined and the string "undefined" gets added to the data attribute in every case. You can check out the snapshots below for an example of the output.

Reply:
yup makes sense - am I understanding the className definition right that the default is "column-list--boxed"? If so I'll use `column-list${plain ? "--plain" : "--boxed"}`

Reply:
Yep, well spotted! Sounds good